e3sm_to_cmip/mpas.py

```diff
@@ -39,20 +73,19 @@ def remap(ds, mappingFileName, threshold=0.05):
     env = os.environ.copy()
     env['NCO_PATH_OVERRIDE'] = 'no'
 
-    args = ['ncremap', '-7', '--dfl_lvl=1', '--no_stdin',
+    if "sgs" in mappingFileName:
+        args = ['ncremap', '-7', '--dfl_lvl=1', '--no_stdin',
             '--no_cll_msr', '--no_frm_trm', '--no_stg_grd', '--msk_src=none',
-            '--mask_dst=none', '--map={}'.format(mappingFileName), inFileName,
-            outFileName]
+            '--mask_dst=none', f'--map={mappingFileName}', inFileName, outFileName]
+        run_ncremap_cmd(args, env)
```

Comment:
I think there's something weird with the if/elif/if going on here.  Presumably, you want to handle:
* mpasseaice and sgs
* mpasseaice without sgs
* mpaso
It seems to me that relying on "sgs" in the mapping file name is a little too fragile.  It would be better to explicitly provide an argument about whether sgs shoudl be use.  There should be an error if you try to use sgs with mpaso, since that isn't supported.  There could be a sanity check and and error or a warning if sgs is requested but the mapping file doesn't contain "sgs" in its name or if sgs is *not* requested and sgs *is* in the name.  But I'll leave this up to you.  What you currently plan is going to work fine as long as the mapping files are named as expected.